sky/shell/BUILD.gn

```diff
@@ -258,6 +258,8 @@ if (is_android) {
     source_set(target_name) {
 
       sources = [
+        "platform/ios/accessibility_bridge.h",
+        "platform/ios/accessibility_bridge.mm",
```

Comment:
I think it depends on whether the file defines an objc class or a c++ class, so I think I'm doing it right here.  Chinmay, yell if I'm wrong :-)